spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/spring-cloud-gcp-integration-pubsub-sample-polling-receiver/src/test/java/com/example/PollingReceiverIntegrationTest.java

```diff
@@ -45,38 +44,33 @@
  *
  * @since 1.2
  */
-
-@RunWith(SpringRunner.class)
+@EnabledIfSystemProperty(named = "it.pubsub-integration", matches = "true")
+@ExtendWith(SpringExtension.class)
 @SpringBootTest
 @DirtiesContext
-public class PollingReceiverIntegrationTest {
+class PollingReceiverIntegrationTest {
 	private static PrintStream systemOut;
 
 	private static ByteArrayOutputStream baos;
 
 	@Autowired
 	private PubSubTemplate pubSubTemplate;
 
-	@BeforeClass
-	public static void prepare() {
-		assumeThat(
-				"PUB/SUB-sample integration tests are disabled. Please use '-Dit.pubsub-integration=true' "
-						+ "to enable them. ",
-				System.getProperty("it.pubsub-integration"), is("true"));
-
+	@BeforeAll
+	static void prepare() {
 		systemOut = System.out;
 		baos = new ByteArrayOutputStream();
 		TeeOutputStream out = new TeeOutputStream(systemOut, baos);
 		System.setOut(new PrintStream(out));
```

Comment:
It should be possible to use `OutputCaptureExtension` now instead of this homemade stuff.

Reply:
Done. Does it LGTY now?